Vagrantfile

```diff
@@ -151,7 +157,7 @@ Vagrant.configure('2') do |config|
   end
 
   # Cruisin' Mos Espa In my Delorean ...
-  (1..3).each do |n|
+  [1].each do |n|
```

Comment:
I should revert this (and my change to the cluster setup)... I did this so I had an indexer cluster of 1 node in Vagrant so I minimized the number of VMs I needed to have running.